worker/src/test/java/org/apache/celeborn/service/deploy/worker/storage/PartitionFilesSorterSuiteJ.java

```diff
@@ -87,6 +83,7 @@ public void prepare(boolean largefile) throws IOException {
                 }
                 return v;
               });
+      // [63.9k, 192k + 63.9k]
```

Comment:
I think this is to make sure that dataSize will not exceed 256k, the default flush threshold.